docs/docs-components/AppLayout.js

```diff
@@ -2,6 +2,10 @@
 import { Fragment, type Node as ReactNode, useEffect, useState } from 'react';
 import { useRouter } from 'next/router';
 import { Box, DeviceTypeProvider, Divider, FixedZIndex, Flex } from 'gestalt';
+import {
+  TOKEN_COLOR_GRAY_ROBOFLOW_700,
+  TOKEN_COLOR_ORANGE_FIRETINI_0,
+} from 'gestalt-design-tokens/dist/js/constants';
```

Comment:
Would it make sense to update the package exports to the js file instead of the css file. I'm not sure if the variables.css file is consumed directly

What if you change `package.json` in design-tokens
```
main": "dist/css/variables.css" -> main": "dist/js/constants.js"
```

So it becomes like this

```
import  {TOKEN_COLOR_GRAY} from `gestalt-design-tokens`
```

Reply:
yep, that´s a good point.

Reply:
DONE